src/App.ts

```diff
@@ -448,17 +454,17 @@ export default class App {
       );
     } else if (authorize === undefined && !usingOauth) {
       throw new AppInitializationError(
-        `No token, no authorize, and no oauth installer options provided. ${tokenUsage}`,
+        `${tokenUsage} \n\nSince you have not provided a token or authorize, you might be missing one or more required oauth installer option(s). See https://slack.dev/bolt-js/concepts#authenticating-oauth for these required fields.\n`,
       );
     } else if (authorize !== undefined && usingOauth) {
-      throw new AppInitializationError(`Both authorize options and oauth installer options provided. ${tokenUsage}`);
+      throw new AppInitializationError(`You provided both authorize and oauth installer options. ${tokenUsage}`);
```

Comment:
Since this is an error message, would it be more clear to say: "You cannot provide both authorize and oauth installer options."? I'm not sure and just asking for other opinions.